static/src/stylesheets/module/_adslot.scss

```diff
@@ -1,7 +1,7 @@
 .ad-slot {
     position: relative;
     z-index: $zindex-ads;
-    overflow: hidden;
+    overflow: initial;
```

Comment:
I understand this one, just slightly worried about side effects. Let's keep an eye when it's out!